core/vulkan/vk_virtual_swapchain/cc/swapchain.cpp

```diff
@@ -15,12 +15,52 @@
  */
 
 #include "swapchain.h"
+
 #include <cassert>
+#include <sstream>
 #include <vector>
+
 #include "virtual_swapchain.h"
 
 namespace swapchain {
 
+// Used to set the value of VkSurfaceCapabilitiesKHR->currentExtent
+// returned from vkGetPhysicalDeviceSurfaceCapabilitiesKHR.
+// E.g. VIRTUAL_SWAPCHAIN_SURFACE_EXTENT="1960 1080"
+// If unset then the current extent will be the "special value"
+// {0xFFFFFFFF, 0xFFFFFFFF}, which some apps don't handle well.
+// I.e. they will try to create a swapchain with this maximum extent size and we
+// will then fail to create a buffer of this size.
+const char* kOverrideSurfaceExtentEnv = "VIRTUAL_SWAPCHAIN_SURFACE_EXTENT";
+// Android property names must be under 32 characters in Android N and below.
```

Comment:
Do we care about Android N and below?

Reply:
I vote yes because Vulkan exists on Android N (and this Vulkan layer can be used outside AGI) and the only requirement is we keep property names under 32 chars, which is not too bad.

Reply:
Sounds good!